.github/workflows/pull-request.yml

```
@@ -432,11 +433,79 @@
+    runs-on: ubuntu-latest
+    needs: [Code-Quality-Checks]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '22'
+      - name: Generate Documentation
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC1049:error:9:1: Did you forget the 'then' for this 'if'?

(shellcheck)